packages/tabs/src/Tabs.tsx

```diff
@@ -132,36 +133,38 @@ function Tabs({
     setSelected(index);
   }
 
-  function handleKeyDown(e: React.KeyboardEvent) {
-    e.stopPropagation();
+  const enabledIndexLogic: () => [Array<number>, number] = () => {
```

Comment:
Most code is logic. Maybe `getEnabledIndexes`?

Reply:
😂 kk